src/Files/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -618,10 +618,11 @@
 
                                 <Grid
                                     Grid.Column="1"
-                                    Width="{Binding ColumnsViewModel.NameColumn.LengthIncludingGridSplitter.Value, ElementName=PageRoot, Mode=OneWay}"
+                                    MinWidth="{Binding ColumnsViewModel.NameColumn.LengthIncludingGridSplitter.Value, ElementName=PageRoot, Mode=OneWay}"
```

Comment:
Can you set the max width to the not be larger than the window width?

Reply:
@yaichenbaum I already tested this with my initial changes, and it works fine. No need for any changes.

Reply:
If it goes wider than the page you currently need to scroll the whole listview, the expected behavior is to scroll in the textbox the same way it works in File Explorer.

Reply:
@yaichenbaum I think this is an improvement regardless.

Reply:
Could you check if d9197d4 solves this issue?

Reply:
Looks good